source/clear-linux/guides/network/firewalls.rst

```diff
@@ -0,0 +1,198 @@
+.. _firewalls:
+
+Firewalls
+#########
+
+Firewalls control access to and from systems based on network packet
+attributes like IP address, port, payload and more. 
+
+The `Netfilter <https://netfilter.org/>`_ framework in the Linux kernel
+performs packet filtering and provides the means for implementing a software
+firewall in Linux. |CL-ATTR| has a couple different firewall front-end options
+for managing the Linux firewall.
+
+
+.. contents:: :local:
+    :depth: 2
+   
+
+Default ruleset
+***************
+
+|CL| does not impose a firewall policy out of the box. All traffic is allowed
+inbound and all traffic is allowed outbound. However, `tallow`_ is installed
+by default and may dynamically create rules temporarily restricting SSH
+access.
+
+.. warning::
+
+   Changing firewall configuration can cause abrupt network disconnection, if
+   misconfigured. If this happens on a remote host, local recovery may be
+   required.
+   
+   Be sure to test your firewall configuration before committing it permanently.
+
+
+iptables 
+********
+
+:command:`iptables` is a well-known user-space administration tool for
+configuring IPv4 Linux firewall rules. :command:`ip6tables` is a
+complimentary tool for configuring IPv6 Linux firewall rules. 
+
+Below is information on using :command:`iptables` on |CL|: 
+
+#. Make sure the *iptables* bundle is installed
+
+   .. code:: bash
+
+      sudo swupd bundle-add iptables
+
+
+#. Define new iptables rules/chains for the running configuration using the
+   :command:`iptables` command. See :command:`man iptables` for iptables
+   concepts and configuration options.
+
+   Below is a common restrictive firewall configuration which denies all
+   incoming connections, unless the connection was initiated by the host. 
+
+   .. code:: bash
+
+      #Delete all existing rules 
+      sudo iptables -F
```

Comment:
I wouldn't do this by default. Definitely note that some programs may insert rules and expect this not to be done randomly.